src/server/plugins/rh_heatgenerator_ladder/__init__.py

```diff
@@ -0,0 +1,205 @@
+'''Standardized heat structures'''
+# FAI: https://www.fai.org/sites/default/files/ciam/wcup_drones/sc4_vol_f9_dronesport_22_2022-03-01_0.pdf
+# MultiGP: https://docs.google.com/document/d/1jWVjCnoIGdW1j_bklrbg-0D24c3x6YG5m_vmF7faG-U/edit#heading=h.hoxlrr3v86bb
+
+import logging
+logger = logging.getLogger(__name__)
+import RHUtils
+import random
+from HeatGenerator import HeatGenerator
+from Database import ProgramMethod
+
+def registerHandlers(args):
+    if 'registerFn' in args:
+        for generator in discover():
+            args['registerFn'](generator)
```

Comment:
Just from reading this, I was very confused as to what `discover` is. While Python doesn't enforce this (other than C/C++), and sometimes cyclic dependencies prevent a global order, in general I prefer things to be ordered from specialized to generic. You read the whole file and get an idea what subsequent things do based on things you saw before, instead of having to mentally park them, and revisit. Lots of taste in this one of course.